cf_units/tests/test_coding_standards.py

```diff
@@ -52,9 +52,9 @@
 
 
 LICENSE_RE_PATTERN = re.escape(LICENSE_TEMPLATE).replace(r'\{YEARS\}', '(.*?)')
-# Add shebang possibility to the LICENSE_RE_PATTERN
-LICENSE_RE_PATTERN = r'(\#\!.*\n)?' + LICENSE_RE_PATTERN
-LICENSE_RE = re.compile(LICENSE_RE_PATTERN, re.MULTILINE)
+SHEBANG = r'(\#\!.*\n)?'
+ENCODING = r'(\# \-\*\- coding\: .* \-\*\-\n)?'
```

Comment:
@pelson You might want to account for white space after the trailing `-*-` ...?

Reply:
Meh. Happy if you want me to, but why be generous on this? Do it right, or fix it is my opinion 😉